astropy/modeling/polynomial.py

```diff
@@ -1391,8 +1395,9 @@ class SIP(Model):
     .. [1] `David Shupe, et al, ADASS, ASP Conference Series, Vol. 347, 2005 <http://adsabs.harvard.edu/abs/2005ASPC..347..491S>`_
     """
 
-    inputs = ('u', 'v')
-    outputs = ('x', 'y')
+    n_inputs = 2
+    n_outputs = 2
+
```

Comment:
Does this change the default name of the inputs since now it uses the default names x, y?

Reply:
Yes, fix coming.